posthog-node/src/feature-flags.ts

```diff
@@ -605,7 +611,9 @@ function matchPropertyGroup(
         }
       } catch (err) {
         if (err instanceof InconclusiveMatchError) {
-          console.debug(`Failed to compute property ${prop} locally: ${err}`)
+          if (debugMode) {
+            console.debug(`Failed to compute property ${prop} locally: ${err}`)
+          }
```

Comment:
See https://github.com/PostHog/posthog-js-lite/pull/219/files#r1579057636
Could be replaced by `console.debug(`Failed to compute property ${prop} locally: ${err}`)` that internally abstracts `debugMode`